swarm/storage/dbstore.go

```diff
@@ -726,19 +686,14 @@ func (s *DbStore) get(key Key) (chunk *Chunk, err error) {
 			hash := hasher.Sum(nil)
 
 			if !bytes.Equal(hash, key) {
-				log.Error(fmt.Sprintf("Apparent key/hash mismatch. Hash %x, key %v", hash, key[:]))
+				log.Trace(fmt.Sprintf("Apparent key/hash mismatch. Hash %x, key %v", hash, key[:]))
```

Comment:
these should be errors actually

Reply:
The Trace log function is but back from https://github.com/ethersphere/go-ethereum/blob/swarm-network-rewrite-syncer/swarm/storage/dbstore.go#L694. I was not aware that it was corrected in swarm-network-rewrite-syncer-test, I will change it.